test/extended/cli/explain.go

```diff
@@ -598,6 +607,25 @@ var _ = g.Describe("[sig-cli] oc explain", func() {
 			}
 		})
 	}
+
+	for group, sets := range specialExceptionTypes {
+		groupName := group
+		types := sets
+		for _, set := range types {
+			resourceName := strings.Split(set.field, ".")[0]
+			g.It(fmt.Sprintf("should contain proper fields description for %s of %s [apigroup:%s]", resourceName, groupName, groupName), func() {
```

Comment:
```suggestion
			g.It(fmt.Sprintf("should contain proper fields description for %s of %s, if the type is present [apigroup:%s]", resourceName, groupName, groupName), func() {
```

Reply:
`type` can confuse between group and resource. How about `if the resource is present` ?